### DISCUSSION TITLE
Pending items

### DISCUSSION THREAD

**Discussion by gargajit** [FifthTry/admint.io#15] [Ideas & RFCs]
Following are the Pending items when [ftd-admint.io](https://fifthtry.github.io/admint.io/index.html) is compared with [admint.io](https://admint.io/#/home):

- [ ] index page
  - [x] page title is not the same
  - [x] fab icon is missing
  - [x] Header section font-weight is not matching
  - [x] Hero component's title font-weight is not matching
  - [x] Hero component's top left image is not pixel matched
  - [x] Slider navigation is out of position
  - [ ] Slider movement animation is not matching
  - [ ] Slider navigation's foreground and the background color is not matching
  - [x] Slider 1.1: title font and font-weight is not matching
  - [x] Slider 1.1: the sub-title is different, quotes are missing, `Web 3.0 ready` text color is wrong
  - [x] Slider 1.1: the `quotes` around `Web 3.0 ready` is not matching
  - [ ] Slider 1.1: numbers 1, 2, and 3 are of different font sizes, it should be `73*73`
  - [x] Slider 1.1: card 3 position is not matching (*I think our version is better: cards consistency is better in our page*)
  - [x] Slider 1,1: the 3 cards are few pixels higher than original (not pixel matched)
  - [ ] Slider 1.2: title font and font-weight is not matching
  - [x] Slider 1.2: the sub-title is different, and `standard known processes:` text color missing
  - [x] Slider 1.2: body font and font-weight is not matching. 
  - [x] Slider 1.2: body placement is different and more words in the first line
  - [x] Slider 1.2: image position is not matching
  - [x] Slider 1.3: title font and font-weight is not matching
  - [x] Slider 1.3: the sub-title is different, `key media vehicle` text color is wrong
  - [x] Slider 1.3: sub-title has more padding with title
  - [x] Slider 1.3: body font and font-weight is not matching. 
  - [x] Slider 1.3: body placement is different and more words in the first line
  - [x] Slider 1.3: image position is not matching
  - [x] Slider 1.4: title font and font-weight is not matching
  - [x] Slider 1.4: sub-title is different, `distribute, validate & track` text color is wrong
  - [x] Slider 1.4: body font and font-weight is not matching
  - [x] Slider 1.4: body text is wrong
  - [x] Slider 1.4: body has more words in the first line
  - [x] Slider 1.4: image position is not matching
  - [x] Slider 1.5: Slider 1.5 do not match at all
  - [x] Slider 1.5: title font and font-weight is not matching
  - [x] Slider 1.5: sub-title is different, `Analyze & consolidate` text color is wrong
  - [x] Slider 1.5: sub-title has more words in the first line
  - [x] Slider 1.5: body font is not matching
  - [x] Slider 1.5: body placement is different and more words in the first line
  - [x] Slider 1.5: half the image is not there. Also, the position is not correct.
  - [x] Slider 1.5: image position is not matching
  - [x] Promo banners for `NFTs` is missing
  - [x] Promo banner (help-banner) text font is wrong. Also, the help-banner dimension is wrong. `1296*295` and ours is `1320*295`
  - [ ] Promo banner (help-banner) for NFTs text `media vehicle` is slightly out of position
  - [ ] Promo banner (help-banner) for NFTs `button` dimensions are different. It should be `171.1*49`
  - [x] Promo banner (help-banner) for NFTs `button` text `Let's Learn` is different 
  - [x] `HOW IT WORKS` and `ADVERTISING IN WEB 3.0` and Promo banner for `Confused about everything Web 3.0?`  are not in the correct order
  - [ ] `HOW IT WORKS` sliders cannot be moved from Step 1 to Step 2 using a mouse movement after holding left-click
  - [ ] `HOW IT WORKS` slider navigation foreground and the background color is not matching
  - [x] Slider 2.1: font of the title `HOW IT WORKS` is not matching
  - [x] Slider 2.1: `HOW IT WORKS` slider sub-title color is wrong
  - [x] Slider 2.1: pixel of `set up made simple` is not matching
  - [x] Slider 2.1: STEP 1 planning texts are not in the right place, the color is wrong, box text is also not in the right place
  - [ ] Slider 2.1: text `STEP 1`, font is not perfectly matching 
  - [x] Slider 2.2: `STEP 2` image is not matching
  - [x] Slider 2.2: Texts font is different inside the box
  - [x] Slider 2.2: Text `AdMint handles....transfers` is out of position
  - [ ] Slider 2.3: text `STEP 3` font not perfectly matching
  - [x] Slider 2.3: image pixel not matching 
  - [x] Slider 2.3: `STEP 3` image is not matching
  - [x] Slider 2.3:  Texts font is different inside the box
  - [x] Slider 2.3:  Text `AdMint preps....collections` is out of position
  - [ ] Slider 2.4: Text `STEP 4` font is not perfectly matching
  - [x] Slider 2.4: Extra word - remove the word `execution` from the box with green background
  - [x] Slider 2.4: image pixels are not matching
  - [ ] Slider 2.5: Text `STEP 5` font is not matching
  - [x] Slider 2.5: Text `Admint helps...access` is out of position
  - [ ] Slider 2.6: Text `STEP 6` font is not perfectly matching
  - [x] `HOW IT WORKS` Slide 6 is missing
  - [x] `ADVERTISING IN WEB 3.0` advt-title font is not matching
  - [ ] advrt-card on-mouse hover animation is not matching
  - [ ] advrt-card body font and pixel not matching
  - [x] Promo banner (help-banner) for `Confused about everything Web 3.0?` must have padding.
  - [ ] Promo banner (help-banner) button font and dimensions not matching
  - [x] Shadow is missing
  - [ ] AdMint UI image rounded border pixel not matching
  - [x] `sneak peek` text color is not there
  - [x] `sneak peek` image is padded from the right
  - [x] `Get Started` button position has more padding than it should have
  - [x] `sneak peek` button  `Get Started` dimensions and font not matching
  - [x] Sign Up form: `waitlist1 text color is wrong
  - [x] Sign Up form: Field names font is not matching
  - [x] Sign Up form: Button's text `Join waitlist` font-weight does not match
  - [ ] Sign Up form: title position is not pixel matched
  - [ ] Sign Up form: font of default value `Advertiser` of the field `Who are you?` is not matching
  - [ ] Sign Up form: the drop down icon font of the field `Who are you?` is not matching
  - [x] footer thumbs-up image is padded too much to the right
  - [x] footer section list's font is not matching
  
- [ ] Why NFT? (work in progress) 

- [ ] About Us
  - [ ] About Admint Title's text color is wrong
  - [ ] Body font is not matching (*I think our version is better. Reason: better readability*)
  - [ ] Background shadow is not matching
  - [ ] About wrapper dimension (height) is not matching. It should be `383`, we have `383.6`. 
  - [ ] Our Team's `Team` text color is wrong
  - [ ] Our Team's image is not pixel-matched (*I think our version is better. Reason: Image is properly displayed*)
  - [ ] Our Team's cards are not pixel-matched

- [ ] Pricing:
- [ ] Mobile View:
   - [ ] Prices are not blur 
   - [ ] Pricing page background should be white
   - [ ] `For Agencies` - vertical alignment inside circle image mismatch
   - [ ] `For Advertisers` color should be `#000000` vice versa for non-selected  `For Agencies`
   - [ ]  Pricing cards title `BUSINESS` etc font size mis-match
   - [ ] Pricing cards height mis-match
- [ ] Desktop View:
   - [ ] Page background should be white
   - [ ] `The right plan for your business plan.` page title position/padding mis-match
   - [ ] `max-width: 1320px` with `12px` horizontal padding
   - [ ]  Circled image size mis-match
   - [ ]  `For Advertisers` color should be `#000000` vice versa for non-selected  `For Agencies`
   - [ ] `For Advertisers` and `For Agencies` `font-weight` should be `500`
   - [ ] spacing between pricing cards should be `24px` currently its `16 + 16`
   - [ ] `$` price font line height should be `60px` 
   - [ ] card `margin-bottom` should be `20px` 

- [ ] Learn
  - [ ] Task box font-weight and color is not matching
  - [ ] Task box selected task is not encircled
  - [ ] Page should have left and right padding (*I think our version is better. Reason: Page is filled properly*)
  - [ ] FAQ section: body font is not matching (*I think our version is better. Reason: better readability*)
  - [ ] Footer is wrong in Learn, Copyright should be displayed.

**Comment by amitu** [comment]
Pending changes from https://github.com/FifthTry/admint.io/pull/18. 

- [ ] Mouse over on drop-down option - not implemented.
- [ ] on mouse-out drop-down list is not closing
- [ ] Join Wait-list Popup should open on-click of all - Get Started button
- [ ] Learn - Page - Tabs - shadow position from left-bottom mis-match

**Mobile View**
- [ ]  Learn - page - tabs font mis-match with https://admint.io/learn/
- [ ]  Tabs position from top - mis-match
- [ ]  FAQ - title - font mis-match
- [ ]  FAQ - body - font mis-match
- [ ]  Home - Join Waitlist form - horizontal spacing mis-match
- [ ]  All form label, input field placeholder - font size mis-match
- [ ]  Drop-down option list - width not matches with field width

**Comment by gargajit** [comment]
Pending changes from https://github.com/FifthTry/admint.io/pull/13

**Desktop View:**

- [ ]  Slide - 1 - image position is wrong, move it to top.
- [ ]  Slide - 2 - WHY ADMINT? text size mis-match
- [ ]  Slide - 2 - image move to left - position mis-match
- [ ]  Slide - 3 - WHY ADMINT? text size mis-match
- [ ]  Slide - 3 - image move to left - position mis-match
- [ ]  Slide - 4 - WHY ADMINT? text size mis-match
- [ ]  Slide - 4 - image move to left - position mis-match
- [ ]  Slide - 5 - WHY ADMINT? text size mis-match
- [ ]  Slide - 5 - image move to left - position mis-match
- [ ]  Check arrows - when disabled - images mis-matching
- [ ]  Left and Right arrows - images size and appearance looks different, correct it.

**Reply by gsalunke** [reply]
@gargajit we have fixed above list, plz review it.
https://fifthtry.github.io/admint.io/

**Comment by SuryanshShahi** [comment]
Promo banners for NFTs is missing (Done)
Promo banner for Confused about everything Web 3.0? must have padding. (Done)
Shadow is missing (Done)
sneak peek text color is not there. Also, the image is padded from the right. (Till now we don't have this feature)
Get Started button position has more padding than it should have. (Done)

**Reply by gargajit** [reply]
@gsalunke  
Following are the updates on these items:
Promo banner (help-banner) though added, the text font is wrong.
Also, (help-banner) dimension is wrong. `1296*295` and ours is `1320*295`

**Comment by priyanka9634** [comment]
@gsalunke I have gone through below points and UI fixes are Done . Please review from your side.

- Header section font-weight is not matching
- Slider navigation is out of position and foreground and background color is not matching
- Slider 1.1: title font and font-weight is not matching
- Slider 1.1: the sub-title is different, quotes are missing, Web 3.0 text color is wrong
-  Slider 1.1: card 3 position is not matching (I think our version is better: cards consistency is better in our page)
- Slider 1.2: title font and font-weight is not matching
- Slider 1.2: the sub-title is different, standard known processes: text color missing
-  Slider 1.2: body font and font-weight is not matching
-  Slider 1.2: Image position is not right
- Slider 1.3: title font and font-weight is not matching
-  Slider 1.3: the sub-title is different, key media vehicle text color is wrong
-  Slider 1.3: body font and font-weight is not matching
- Slider 1.3: Image position is not right
- Slider 1.4: title font and font-weight is not matching
-  Slider 1.4: the sub-title is different, distribute, validate & track text color is wrong
- Slider 1.4: body font and font-weight is not matching. Also, the body text is wrong
- Slider 1.4: Image position is not right
- Slider 1.5: Slider 1.5 do not match at all
- Slider 1.5: title font and font-weight is not matching
- Slider 1.3: the sub-title is different, Analyze & cons  text color is wrong
-  Slider 1.3: body font and font-weight is not matching
- Slider 1.3: Half the image is not there. Also, the position is not correct.
- HOW IT WORKS and ADVERTISING IN WEB 3.0 and Promo banner for Confused about everything Web 3.0? are not in the correct order
-  `HOW IT WORKS' slider sub-title color is wrong
-  HOW IT WORKS Step 1 planning texts are not in the right place, the color is wrong, box text is also not in the right place
- ~HOW IT WORKS Slide 6 is missing~ -This is not Done
- HOW IT WORKS navigation foreground and background color is not matching

PR 13:

 Slide - 1 - image position is wrong, move it to top.
 Slide - 2 - WHY ADMINT? text size mis-match
 Slide - 2 - image move to left - position mis-match
 Slide - 3 - WHY ADMINT? text size mis-match
 Slide - 3 - image move to left - position mis-match
 Slide - 4 - WHY ADMINT? text size mis-match
 Slide - 4 - image move to left - position mis-match
 Slide - 5 - WHY ADMINT? text size mis-match
 Slide - 5 - image move to left - position mis-match
 Check arrows - when disabled - images mis-matching
 Left and Right arrows - images size and appearance looks different, correct it.

**Reply by gargajit** [reply]
@priyanka9634 I will keep adding/updating this comment section with my review comments. 
Note: For the ones you said are done, and I find it to be done, I will check the box else I will comment here, pending, and leave the checkbox unchecked.

- The header section (navbar) Section font-weight is still not matching.
- Slider 1.1: title font is bold
- Slider 1.1: quotes are missing, inline color will be done later
-  Slider 1.2: title font is bold

**Comment by gsalunke** [comment]
@gargajit Please, review below points, I have gone through these:
Individual task owners have gone through their tasks and my review is pending. Will keep reviewing meanwhile you can, start your review.

1.  Why NFT? - live
2. About Us
- ~About Admint Title's text color is wrong~ - apart from green color, color code used is same.
-  ~Body font is not matching~ (I think our version is better. Reason: better readability) - Body font updated as per admint.io specs.
-  ~Background shadow is not matching~ - Updated.
-  ~Our Team's Team text color is wrong~ -  - apart from green color, color code used is same.
-  Our Team's image is not pixel-matched (I think our version is better. Reason: Image is properly displayed)
-  Our Team's cards are not pixel-matched

3. Learn

- Task box font-weight and color is not matching
- Task box selected task is not encircled
- Page should have left and right padding (I think our version is better. Reason: Page is filled properly)
- FAQ section: body font is not matching (I think our version is better. Reason: better readability)
- Footer is wrong in Learn, Copyright should be displayed.

**Comment by gsalunke** [comment]
@gargajit we have fixed index page and is reviewed by me. Please, review from your side.
https://fifthtry.github.io/admint.io/

**Reply by gargajit** [reply]
@gsalunke Can you correct following mismatched positioning.

1. Hero component

- [x] Both Images 
- [x] sub-title position 
![PerfectPixel-1](https://user-images.githubusercontent.com/118595104/230866936-9fe75cfc-187b-458d-a36d-954c568dabdf.png)

2. Carousel 1 (Slider 1)

- [x] The left carousel icon at slide 1 should have faded arrow
- [x] The right carousel icon's position
- [ ] Font size of numericals

![PerfectPixel-2](https://user-images.githubusercontent.com/118595104/230867067-aaa152a7-2c90-47ae-a6f4-a33e27f44417.png)

3. Carousel 2 (Slider 2)

- [x] Positioning of images and Step 1 with title and sub-title
- [ ] Positioning of carousel icons

![PerfectPixel-3](https://user-images.githubusercontent.com/118595104/230868030-cfe74d7f-1488-466c-8233-873b0a00f560.png)

4. Carousel 2 (Slider 2)

- [x] Right carousel icon
![PerfectPixel-4](https://user-images.githubusercontent.com/118595104/230868339-4c91117c-9823-4c1c-9990-e255a7d0a697.png)

5. Card

- [ ] Body text
![PerfectPixel-5](https://user-images.githubusercontent.com/118595104/230868505-3e16f0a9-90a5-48ee-9fce-04343ca6f9e3.png)

6. Sneek Peak 

- [ ] image position
![PerfectPixel-6](https://user-images.githubusercontent.com/118595104/230868713-108f82de-acf1-4c4c-85db-0efbc5c45905.png)

7. Sign up form

- [x] image position
- [ ] input field position
- [ ] drop down arrow font is different

8. Footer

- [x] Image position
![PerfectPixel-8](https://user-images.githubusercontent.com/118595104/230869066-acafd7af-5f72-43ca-8571-88f8742197a9.png)


Process followed:

1. Took screenshot of entire Home from [admint-live](https://admint.io/#/) page using [Fireshot plugin](https://chrome.google.com/webstore/detail/take-webpage-screenshots/mcbpblocgmgfnpjjppndjkmgjaogfceg?hl=en)
2. Opened [fifthtry-admint](https://fifthtry.github.io/admint.io/)
3. Used [Perfect pixel](https://chrome.google.com/webstore/detail/perfectpixel-by-welldonec/dkaagdgjmgdmbnecmcefdhjekcoceebi?hl=en) extension
4. For position match: Set Opacity to 50% and comapared, position
5. For finding difference: keep on sliding Opacity meter between 0% to 100% 

Note: Grammarly extension settings have been changed to work only when extension is selected. Did not affect the pages.

**Reply by gsalunke** [reply]
@gargajit since these issues were seen on your machine before `Overlay Scrollbars` chrome testing issue. Can you plz cross check these and check if these are matching?

**Reply by gargajit** [reply]
@gsalunke I have checked which are matching after enabling `Overlay Scrollbars` using `chrome://flags/` via browser search bar. The remaining unchecked are still there.

**Comment by gargajit** [comment]
Apart from the [Known Issues](https://github.com/FifthTry/admint.io/discussions/91). Following are some minor mismatch that needs a fix.

**Process followed:**

1. Overlay Scrollbars was `enabled` via `chrome://flags/`, as requested by Ganesh. (It helps to remove the scrollbar 
2. Took screenshot of entire Home from [admint-live](https://admint.io/#/) page using [Fireshot plugin](https://chrome.google.com/webstore/detail/take-webpage-screenshots/mcbpblocgmgfnpjjppndjkmgjaogfceg?hl=en)
3. Used [Perfect pixel](https://chrome.google.com/webstore/detail/perfectpixel-by-welldonec/dkaagdgjmgdmbnecmcefdhjekcoceebi?hl=en) extension to compare the screenshot with [fifthtry-admint](https://fifthtry.github.io/admint.io/)
4. Following are the settings used for testing:
X: 0
Y: 0
Scale: 0.8
Opacity: 50% (Opacity was changed between 0-100%, for finding any pixel difference)
![image](https://user-images.githubusercontent.com/118595104/231075414-33838638-4502-4e99-b998-47214e7efc13.png)


1. Hero: 

- [ ] enclosing `circle` for text Web 3.0
- [ ] Position of `Get Started` button


2. Carousel 1

- [ ] Sub-title position
- [ ] Card border contrast is sharper than live
- [ ] center-align of card numbers (discussed)

3. Banner 1:

- [ ] `Let's Learn` button font and it's contrast (Ours is better. Reason, better contrast)
- [ ] Button dimensions are still not matching. Dimensions in live site are `171.1*49`
![image](https://user-images.githubusercontent.com/118595104/231105958-cb267766-4293-4f8b-8ab6-7a90fb055758.png)


4. Carousel 2:

- [ ] Step, box and content not pixel matched (Please do the same for other steps in following sliders)
![image](https://user-images.githubusercontent.com/118595104/231080741-85b37ec8-ccf4-44cc-be4a-f9f663dc1902.png)
![image](https://user-images.githubusercontent.com/118595104/231113934-f17bf6d3-9390-4150-99d6-c877a6d9379d.png)


5. Advertisement Card:

- [ ] title and sub-title 
- [ ] Body text
![image](https://user-images.githubusercontent.com/118595104/231083417-795d9a35-6307-4a9e-8948-6288e223d90e.png)


6. Banner 2:

- [ ] text and image not pixel matched
- [ ] `Let's Learn` button font-weight(expected 600, but its 700) and it's contrast
![image](https://user-images.githubusercontent.com/118595104/231109124-eadfcb0c-aecd-4cf7-9e74-d1081324769b.png)

![image](https://user-images.githubusercontent.com/118595104/231084383-f5b29327-7ef6-418b-8728-e6b48b7dbcc6.png)

7. Sneak Peek

- [ ] Positions of Image, title, image, Button not pixel matched.
![image](https://user-images.githubusercontent.com/118595104/231085121-629e76ab-a8aa-4307-ba21-a2ed495d976a.png)

- [ ] Image top and bottom border-width is slightly different 
- [ ] Button text font and it's contrast (Ours is better. Reason, better contrast)

 
8. Sign-up form

- [ ] Position not matching
![image](https://user-images.githubusercontent.com/118595104/231086125-823b836d-fcef-4377-ad3b-9810d47aae06.png)

- [ ] Field value `advertiser` font is not matching
- [ ] drop-down icon font not matching

9. Footer

- [ ] Footer with setting values (X,Y,Scale) is not matching. I had to change the Y axis value to 5
![image](https://user-images.githubusercontent.com/118595104/231087050-374598a7-c3d2-40f1-bd00-ffd32e192b7c.png)

**Reply by gsalunke** [reply]
@gargajit Below is the comparison with original layer of admint.io. This is the final draft we are sending for your review:


![home-page-px-to-px-11-April-2023](https://user-images.githubusercontent.com/68585007/231174389-d5565a3d-d61e-4fc4-8004-daebd42329be.png)

**Comment by gsalunke** [comment]
@gargajit from list of top
Cross-checked below item:

 Slider 1.1: numbers 1, 2, and 3 are of different font sizes, it should be 73*73

<img width="504" alt="image" src="https://user-images.githubusercontent.com/68585007/231131225-0c555ff4-5461-46ed-b7fd-f8acee131213.png">


As per above screen-shot we can see its 73 x 73. Please, cross check.
